### PR TITLE
step for dns basedomain

### PR DESCRIPTION
**Pull request by lihongan**
helper for openshift/cucushift#6991

@akostadinov @pruan-rht pelase help review.

### CODE REVIEW

File: features/step_definitions/route.rb [hunk 1]
```diff
@@ -131,3 +131,13 @@
       | #{label_filter} |
     })
 end
+
+Given /^dns basedomain is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
```
Comment (akostadinov) [inline features/step_definitions/route.rb]: We already have a step `I store default router subdomain in the#{OPT_SYM} clipboard` and it does NOT require admin access. Better use it. It requires you to have a project though. Shouldn't be a big deal comapred to requirement to have admin access.

Comment (lihongan) [reply features/step_definitions/route.rb]: I'd like to get the basedomain here, but actually the subdomain is `"apps." + basedomain` so it is OK for me to use it. Thanks.

**Comment by lihongan** [comment]
I'm closing the PR since we have another similar step.

File: features/step_definitions/route.rb (hunk 1 above)
Comment (akostadinov) [reply features/step_definitions/route.rb]: In such case it is good to create your step , but lets base it on the existing step.

**Pull request closed without merging**